### PR TITLE
Mark a couple items as complete in the roadmap

### PR DESCRIPTION
**Pull request by benmccann**

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -18,30 +18,30 @@ clustered database or key-value store. We will target such workloads for our
...
   - Status: in progress
 6. Error reporting: Report common problems in ways that users can discover
   - Status:
 7. Event management: Make events usable and useful
   - Status:
-8. Persistent storage support (#4055)
+8. Persistent storage support ([#5105](https://github.com/GoogleCloudPlatform/kubernetes/issues/5105))
   - Status: in progress
-9. Allow nodes to join/leave a cluster (#2303,#2435)
+9. Allow nodes to join/leave a cluster ([#2303](https://github.com/GoogleCloudPlatform/kubernetes/issues/2303),[#2435](https://github.com/GoogleCloudPlatform/kubernetes/issues/2435))
```
Comment (roberthbailey) [inline docs/roadmap.md]: Change this link to #6087.

Comment (roberthbailey) [inline docs/roadmap.md]: You can remove #2435. If you want to link a second issue, #3168 would be the other candidate.

File: docs/roadmap.md [hunk 2]
```diff
@@ -18,30 +18,30 @@ clustered database or key-value store. We will target such workloads for our
...
   - Status: high level [design doc](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/clustering.md).
 10. Handle node death
   - Status: mostly covered by nodes joining/leaving a cluster
-11. Allow live cluster upgrades (#2524)
+11. Allow live cluster upgrades ([#2524](https://github.com/GoogleCloudPlatform/kubernetes/issues/2524))
```
Comment (roberthbailey) [inline docs/roadmap.md]: Cluster upgrade is being tracked in #6075 (master) and #6079 (node).

File: docs/roadmap.md [hunk 3]
```diff
@@ -18,30 +18,30 @@ clustered database or key-value store. We will target such workloads for our
...
 6. Error reporting: Report common problems in ways that users can discover
   - Status:
 7. Event management: Make events usable and useful
   - Status:
-8. Persistent storage support (#4055)
+8. Persistent storage support ([#5105](https://github.com/GoogleCloudPlatform/kubernetes/issues/5105))
   - Status: in progress
-9. Allow nodes to join/leave a cluster (#2303,#2435)
+9. Allow nodes to join/leave a cluster ([#2303](https://github.com/GoogleCloudPlatform/kubernetes/issues/2303),[#2435](https://github.com/GoogleCloudPlatform/kubernetes/issues/2435))
   - Status: high level [design doc](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/clustering.md).
```
Comment (roberthbailey) [inline docs/roadmap.md]: This status is now in progress (see #6949).

**Comment by benmccann** [comment]
@roberthbailey thanks! i've updated those items as well

**Comment by roberthbailey** [comment]
Thanks @benmccann. I'm assigning to @brendandburns to take a second look. /cc @thockin and @bgrant0607 as well.

**Comment by bgrant0607** [comment]
Thanks much. We were planning to make another pass over the roadmap soon, also.

File: docs/roadmap.md [hunk 4]
```diff
@@ -18,30 +18,30 @@ clustered database or key-value store. We will target such workloads for our
...
+  - Status: DONE. Released in 0.15.0
+3. Nominal services for applications which need one stable IP per pod instance ([#260](https://github.com/GoogleCloudPlatform/kubernetes/issues/260))
   - Status: #2585 covers some design options.
-4. API input is scrubbed of status fields in favor of a new API to set status (#4248)
-  - Status: in progress
-5. Input validation reporting versioned field names (#2518)
-  - Status: in progress
+4. API input is scrubbed of status fields in favor of a new API to set status ([#4248](https://github.com/GoogleCloudPlatform/kubernetes/issues/4248))
+  - Status: DONE
+5. Input validation reporting versioned field names ([#2518](https://github.com/GoogleCloudPlatform/kubernetes/issues/2518))
```
Comment (nikhiljindal) [inline docs/roadmap.md]: I dont think that this is done yet: https://github.com/GoogleCloudPlatform/kubernetes/issues/3084

**Pull request merged**